src/main/java/com/software/modsen/passengermicroservice/services/PassengerService.java

```diff
@@ -0,0 +1,16 @@
+package com.software.modsen.passengermicroservice.services;
+
+import com.software.modsen.passengermicroservice.entities.Passenger;
+import com.software.modsen.passengermicroservice.entities.PassengerDto;
+import com.software.modsen.passengermicroservice.entities.PassengerPatchDto;
+
+import java.util.List;
+
+public interface PassengerService {
```

Comment:
When I had an interview with a technical specialist and we discussed the solid principle Dependency Inversion, he said that the upper modules should not depend on the lower modules. And as I understand it, the lower module is a service. Therefore, I decided to abstract from the implementation. I also believed that this would help to scale the code in the future.

Reply:
You will fail DI principle if your service layer will use SQL queries, for example